apps/web/components/booking/CancelBooking.tsx

```diff
@@ -99,6 +105,11 @@ export default function CancelBooking(props: Props) {
 
                   if (res.status >= 200 && res.status < 300) {
                     // tested by apps/web/playwright/booking-pages.e2e.ts
+                    sdkActionManager?.fire("bookingCancelled", {
+                      bookingUid: booking?.uid,
```

Comment:
should we add more booking information like for creating/rescheduling booking?

Reply:
Okay